docs/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+Fixes P4-<TODO>
+
+## Proposed Changes
+
+- TODO
+
+## To test/demo change
+
+- TODO
+
+## Manual steps to deploy/dependencies
+
+- TODO (or remove)
```

Comment:
Nice the idea of the template, I also would add the following:

```
## Guidance to review

## Link to Jira card

## Things to check
- [ ] API docs has been updated if necessary
- [ ] New environment variables have been added to staging configuration 
```

Reply:
🙌  I've added `Things to check`. The JIRA issue is the first line o the template, and I think "guidance to review" is covered by "steps to test" (though we could rename that to make it clearer too)